extensions/blocks/calendly/index.js

```diff
@@ -15,6 +16,7 @@ import icon from './icon';
  */
 import './editor.scss';
 
+export const REGEX = /(^|\/\/)(calendly\.com[^"']*)/i
```

Comment:
Tests appear to be failing right now, that should fix them.

```suggestion
export const REGEX = /(^|\/\/)(calendly\.com[^"']*)/i;
```